addons/nuxeo-retention/nuxeo-retention-core/src/main/resources/OSGI-INF/retention-audit.xml

```diff
@@ -1,5 +1,15 @@
 <component name="org.nuxeo.retention.audit" version="1.0">
   <extension target="org.nuxeo.ecm.platform.audit.service.NXAuditEventsService" point="event">
     <event name="retentionRuleAttached" />
+    <event name="afterSetRetention">
+      <extendedInfos>
+        <extendedInfo expression="${message.properties.retainUntil}" key="retainUntil" />
+      </extendedInfos>
+    </event>
+    <event name="afterExtendRetention">
+      <extendedInfos>
+        <extendedInfo expression="${message.properties.retainUntil}" key="retainUntil" />
+      </extendedInfos>
+    </event>
```

Comment:
Should we have the same thing for the before events?

Reply:
No, currently I split into the the before event but in the audit we need only the after onces. and this contribution will be used to add information to the extend property in the audit object.